tests/test_imports.py

```diff
@@ -0,0 +1,93 @@
+"""Import tests."""
```

Comment:
I'd rephrase that since that could also be understood as "We import tests here", while it means "We test imports here".

Reply:
I see what you mean. Changed it 👍🏼